### PR TITLE
File when open file

### LINKED ISSUE #21: Can not open files in WSL2 sessions
When connected to a WSL2 issue I can browse directories but trying to open a file gives me 

`[UriError]: If a URI does not contain an authority component, then the path cannot begin with two slash characters ("//")`

I need this extension so badly, I'm half tempted to try and learn how to code visual studio extensions :). Many thanks for what you've done so far!

VSC 1.67.2 running on Windows 11, accessing WSL2

### PR DESCRIPTION
**Pull request by qiyan2019**
fix #21

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+DISABLE_ESLINT_PLUGIN=true
```
Comment (shirou) [inline .env]: Could you delete this `.env` file?

**Review by shirou — CHANGES_REQUESTED** [review]
LGTM! Could you change one comment? After that I will merge and release new version. Thank you for your contribution.

File: .env (hunk 1 above)
Comment (qiyan2019) [reply .env]: Done

**Review by shirou — APPROVED** [review]
Thank you so much!

**Pull request merged**